src/types/group.ts

```diff
@@ -5,11 +5,13 @@ class Group {
   id: string
   name: string
   path: string
+  attributes: { [prop: string]: any }
```

Comment:
i would declare and export a `RealmGroupAttributes` type in this file (even if just as an alias for `any`) to use both here and in the api functions to reduce tacit duplication of those type defs.